tests/scancode/data/help/help.txt

```diff
@@ -114,6 +114,7 @@ Options:
                              contain over 90% of source files as children and
                              descendants. Count the number of source files in a
                              directory as a new source_file_counts attribute
+    --package-summary        Generate Package Level summary
```

Comment:
This help text needs to be more descriptive. Keep in mind you also need to add documentation sections for this CLI section, but this can be done at a later stage too, when the code is ready.